core/src/test/resources/configs/xsite/bridge.xml

```diff
@@ -29,7 +29,11 @@
            <!--break_on_coord_rsp="true"/>-->
     <MERGE3 max_interval="5000"
             min_interval="3000"/>
-    <FD_SOCK bind_addr="127.0.0.1"/>
+    <FD_SOCK/>
+    <FD_ALL3 timeout="3000"
+             interval="1000"
+    />
+    <VERIFY_SUSPECT timeout="1000"/>
```

Comment:
So with FD_ALL3 it suspects the member first? I thought suspecting members was tossed out of JGroups a while back?

Reply:
Not sure what you mean @wburns, it was always JGroups' responsibility to suspect members. X-site backups have their own rules for taking sites offline, but a site can have 0..N site masters, all of which join the bridge cluster. 

OTOH the bridge configuration already had {{FD_SOCK}}, which suspects a node once it's stopped, and tests don't remove {{FD*}} protocols from the bridge stack based on the {{TransportFlags}}, so having {{FD_ALL3}} might just complicate debugging.

True,  we have any xsite tests using {{DISCARD}} on the site masters, so {{FD_ALL3}} and {{VERIFY_SUSPECT}} aren't really needed, but they

Reply:
Removed